entity-service/src/test/kotlin/com/egm/stellio/entity/service/IAMListenerTests.kt

```diff
@@ -42,10 +42,31 @@ class IAMListenerTests {
             entityService.createEntity(
                 match {
                     it.id == "urn:ngsi-ld:User:6ad19fe0-fc11-4024-85f2-931c6fa6f7e0".toUri() &&
-                        it.properties.size == 1 &&
-                        it.properties[0].compactName == "username" &&
-                        it.properties[0].instances.size == 1 &&
-                        it.properties[0].instances[0].value == "stellio"
+                        it.properties.size == 3 &&
+                        it.properties[2].compactName == "username" &&
```

Comment:
try to make your code more robust by not hardcoding a specific index in the list